ros2launch/ros2launch/command/launch.py

```diff
@@ -103,6 +102,26 @@ def add_arguments(self, parser, cli_name):
             help="Arguments to the launch file; '<name>:=<value>' (for duplicates, last one wins)")
         arg.completer = SuppressCompleterWorkaround()
 
+        sec_args = parser.add_argument_group(title='Security', description='Security related args')
+        arg = sec_args.add_argument(
+            '--secure',
+            metavar='keystore',
+            nargs='?',
+            const='',
+            help=('Launch using ROS 2 security features using specified keystore dir.'
+                  'Will set up a keystore if one is not specified.'),
+        )
+        try:
+            arg.completer = DirectoriesCompleter()  # argcomplete is optional
+        except NameError:
+            pass
+        sec_args.add_argument(
+            '--no-create-keystore',
+            dest='create_keystore',
+            action='store_false',
+            help='Disable creating keystore automatically'
```

Comment:
Doesn't flow quite right, how about:

```suggestion
            help='Disable automatic keystore creation'
```

Reply:
c40721f